src/core/chart-registry.ts

```diff
@@ -173,15 +149,8 @@ export function filterAll (group: string): void {
  * @param {String} [group]
  * @return {undefined}
  */
-export function refocusAll (group: string): void {
-    const charts = chartRegistry.list(group);
-    for (let i = 0; i < charts.length; ++i) {
-        // @ts-ignore
-        if (charts[i].focus) {
-            // @ts-ignore
-            charts[i].focus();
-        }
-    }
+export function refocusAll (group?: string): void {
```

Comment:
Another function which is unused internally; I forgot it existed! 

We should improve our test coverage sometime.

Reply:
I will see that we can add coverage test, it will good to know.

Reply:
Created #1731